lib/rack/tracker/zanox/template/zanox.erb

```diff
@@ -0,0 +1,26 @@
+<% if mastertag %>
+
+<div class="zx_<%= mastertag.id %> zx_mediaslot">
+  <script type="text/javascript">
+    window._zx = window._zx || [];
+    window._zx.push({"id": "<%= mastertag.id %>"});
+    (function(d) {
+      var s = d.createElement("script"); s.async = true;
+      s.src = (d.location.protocol == "https:" ? "https:" : "http:") + "//static.zanox.com/scripts/zanox.js";
+      var a = d.getElementsByTagName("script")[0]; a.parentNode.insertBefore(s, a);
+    }(document));
+  </script>
+</div>
+
+<% end %>
+
+<% tracking_events.each do |event| %>
+
+<script type="text/javascript" src="https://ad.zanox.com/ppl/?<%= options[:account_id] %>&mode=[[1]]&<%= event.write %>">
```

Comment:
yes, it is fixed. Or more precise: No option which the one who integrate that snipped should change.